test_import.py

```
@@ -0,0 +1,37 @@
+"""
+Test script to verify flight-analysis package installation
+"""
+
+import sys
+import os
+
+# Add the src directory to the path so we can import the package
+sys.path.append('src')
+
+try:
+    from google_flight_analysis.scrape import *
```

Ruff (0.12.2)

`from google_flight_analysis.scrape import *` used; unable to detect undefined names

(F403)